R/google_analytics.R

```diff
@@ -77,15 +77,15 @@ getGoogleAnalytics <- function(tableId, lastNDays = 30, dimensions, metrics, tok
     startDate = as.character(lubridate::floor_date(lubridate::today(),"week") - lubridate::weeks(1));
     endDate = as.character(lubridate::today() - 1);
   } else if (dateRangeType == "lastMonthToYesterday") {
-    startDate = as.character(lubridate::floor_date(lubridate::today(), "month") - months(1));
+    startDate = as.character(lubridate::floor_date(lubridate::today(), "month") %m-% months(1));
     endDate = as.character(lubridate::today() - 1);
   } else if (dateRangeType == "lastYearToYesterday") {
-    startDate = as.character(lubridate::floor_date(lubridate::today(), "year") - lubridate::years(1));
+    startDate = as.character(lubridate::floor_date(lubridate::today(), "year") %m-% lubridate::years(1));
     endDate = as.character(lubridate::today() - 1);
   }
 
   if(is.null(startDate)) {
-    startDate <- as.character(lubridate::today() - lubridate::days(lastNDays))
+    startDate <- as.character(lubridate::today() %m-% lubridate::days(lastNDays))
```

Comment:
For days we can use -.

Reply:
thank you for catching it. changed it to use `-`